javascript/javascript3/week2/promises.js

```diff
@@ -0,0 +1,90 @@
+
+//Ex 2 :Promise that resolves after set time
+//creating a function that returns a promise
+function returnsPromise(resolveAfter) {
```

Comment:
You can give this function a more meaningful name like `delay`

Reply:
changed it